web/locales/de.json

```
@@ -134,7 +134,7 @@
         "mapArrows-answer": "Die Pfeile zwischen den Regionen zeigen den physischen Stromfluss (Importe und Exporte) der Elektrizität: Je schneller sie blinken, desto stärker ist der grenzüberschreitende Stromaustausch. Der Elektrizitätaustausch ist von großer Bedeutung, da beim Import von Strom bilanziell auch auch die Kohlenstoffemissionen importiert werden.",
         "mapSolarWindButtons-question": "Wofür sind die “Sonne” und “Wind”-Buttons auf der Karte da?",
         "mapSolarWindButtons-answer": "Die Knöpfe aktivieren die Anzeige der Echtzeit-Windgeschwindigkeit und Sonneneinstrahlung, womit das momentane Potenzial von Photovoltaik- und Windenergieanlagen abgeschätzt werden kann. Das tatsächliche Potenzial dieser Anlagen ist von vielen weiteren Faktoren abhängig, daher dient diese Visualisierung als ungefährer Indikator des Potenzials.",
-        "mapNuclearColors-question": "Warum werden Regionen mit Atomkraftwerken auf der Karte grün dargestellt?",
+        "mapNuclearColors-question": "Warum werden Regionen mit Kernkraftwerken auf der Karte grün dargestellt?",
         "mapNuclearColors-answer": "Die Erzeugung von Strom mittels Kernkraft hat eine niedrige Kohlenstoffintensität (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\") Quelle</a>). Regionen mit hohem Atomstromanteil werden daher auf der Karte grüner dargestellt. Dies bedeutet aber nicht, dass es keine ökologischen Herausforderungen im Zusammenhang mit der Kernenergie gibt (wie dies auch bei anderen Formen der Energieerzeugung der Fall ist). Diese ökologischen Auswirkungen sind nicht Gegenstand dieser Karte.",
```

```suggestion
        "mapNuclearColors-answer": "Die Erzeugung von Strom mittels Kernkraft hat eine niedrige Kohlenstoffintensität (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\") Quelle</a>). Regionen mit hohem Kernstromanteil werden daher auf der Karte grüner dargestellt. Dies bedeutet aber nicht, dass es keine ökologischen Herausforderungen im Zusammenhang mit der Kernenergie gibt (wie dies auch bei anderen Formen der Energieerzeugung der Fall ist). Diese ökologischen Auswirkungen sind nicht Gegenstand dieser Karte.",
```

In the same spirit as the rest of the changes in this PR, maybe `Atomstromanteil` -> `Kernstromanteil` would make sense?

---

hmm, when I think twice about it....Kernstrom ist not really a common word in germany. "Strom aus Kernenergie" would be better I think. But it will make the sentence a little bit longer.

---

Seems like @alixunderplatz and you are on the same page here - sorry for pulling you in the wrong direction with my broken German :smile: